### ISSUE TITLE
Dark Sand doesn't look right - Shifting Sand Land

### ISSUE THREAD

**Issue by SameytheHedgehog** [sm64pc/sm64ex#368]
The sand overlay that's supposed to be over the dark sand is barely noticeable, and you have to concentrate really hard to see it.

Here's what it looks like currently
![Screenshot from 2020-07-06 21-02-29](https://user-images.githubusercontent.com/45528572/86727277-e7c93180-bff0-11ea-8696-5786be1f0e5d.png)

And here's what it's supposed to look like (taken using Mupen64)
![Screenshot from 2020-07-07 20-23-56](https://user-images.githubusercontent.com/45528572/86863057-0af6ee00-c090-11ea-8d4d-5a0d4d066e26.png)

OS: Ubuntu 20.04
Browser: Firefox 78.0.1 (64-bit)

**Comment by HelmicNewciv** [comment]
Are you sure?  I remember as a kid that dark sand being REALLY dark, I don't remember it ever being as light as it was in that second screenshot.  It's hard to find any screenshots of the game being played on a real N64, though I did find screenshots of the DS remake that also have the dark-colored sand - all the other screenshots show that lighter color.  Do we know whether Mupen64 has the color right?

[IGN walkthrough with screenshots from the DS remake.](https://www.ign.com/wikis/super-mario-64-ds/Shifting_Sand_Land)

**Comment by SameytheHedgehog** [comment]
Yeah, I was having trouble finding screenshots of the actual game on hardware so I was basing it off of Mupen64.
Looking at Project64, it has the dark sand too..

**Comment by SameytheHedgehog** [comment]
Ok, so I haven't tracked down a screenshot, but I found some footage of someone doing a playthrough on real hardware:
![image](https://user-images.githubusercontent.com/45528572/93994612-b2adbf00-fd55-11ea-90c9-c4c8d11740b2.png)

the sand matches that of Mupen64's screenshot
source: https://youtu.be/v-Z3MoRoH74?t=365

**Comment by SameytheHedgehog** [comment]
I also booted up my copy of Super Mario 64 DS and the sand was dark there, which is pretty interesting.

**Comment by HelmicNewciv** [comment]
This is weird.  I _presumably_ was playing the US N64 version as a kid, but again distant memories aren't necessarily the most accurate.  Are there differences based on region?  I got the dark sand when I compiled using a US ROM on Manjaro.  What version do speedrunners typically use?

**Comment by SameytheHedgehog** [comment]
As far as I remember from the last time I checked tcrf, the sand doesn't change between versions. Just Marios voice and them fixing the BLJ
